src/components/FindWalletProductTable/index.tsx

```diff
@@ -0,0 +1,78 @@
+import { useMemo, useState } from "react"
+
+import { FilterOption } from "@/lib/types"
+
+import WalletSubComponent from "@/components/FindWalletProductTable/components/WalletSubComponent"
+import { WalletColumns } from "@/components/FindWalletProductTable/data/WalletColumns"
+import { WalletFilters } from "@/components/FindWalletProductTable/data/WalletFilters"
+import { WalletPersonaPresets } from "@/components/FindWalletProductTable/data/WalletPersonaPresets"
+import ProductTable from "@/components/ProductTable"
+
+import { DEFAULT_LOCALE } from "@/lib/constants"
+
+const FindWalletProductTable = ({ wallets }) => {
+  const walletPersonas = WalletPersonaPresets()
+  const walletFilterOptions = WalletFilters()
```

Comment:
Ill change this for the wallet personas, but since the walletfilters have react components in it I think that should be fine as it is. Let me know if you disagree.

Reply:
Actually, im getting error warnings for both because they use the useTranslation hook. Let me know what youd prefer here.

Reply:
Yeah, agree these should be hooks with `useWhatever` naming... As hooks, I would probably also pull them out of the `data` folder and probably just have them sit next to index.tsx